### DISCUSSION TITLE
.arb files are another kind of .erb file

### DISCUSSION THREAD

**Discussion by NewAlexandria** [community/community#19002] [Other Feature Feedback, Questions, & Ideas]
Hello, heads up that `.arb` files are the same as `.erb. files — a mixed ruby + HTML format.  

The `a` in .`arb` is for Active Admin.   

Thanks if someone can add it to the list.

**Comment by IvailoPetrov06** [comment]


**Reply by NewAlexandria** [reply]
Is it a public list? I'd like to follow for when it's deployed.
